solutions/platinum/cf-1254D.mdx

```diff
@@ -0,0 +1,142 @@
+---
+id: cf-1254D
+source: CF
+title: Tree Queries
+author: Rameez Parwez
+---
+
+[Official Editorial (C++)](https://codeforces.com/blog/entry/71594)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O} ((N + Q) \log(N))$
+
+<LanguageSection>
+<CPPSection>
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+const int N = 150001, mod = 998244353;
```

Comment:
capitalize constants

Reply:
and be more descriptive (MAX_N instead of N)

Reply:
done.

Reply:
unresolving this: not done?